examples/hello_world/gcp/config.textproto

```diff
@@ -0,0 +1,16 @@
+# proto-file: oak/proto/manager.proto
```

Comment:
This should probably be under the example folder, one level up. cc @ipetr0v

Reply:
All config files will be under `examples/${NAME}/config` after https://github.com/project-oak/oak/pull/474 is merged.
Also, I think this config file should be called `gcp_config.textproto` since `config.textproto` will be used by `run_examples`.